distribution/docs/src/main/jdocs/content/_quickstart/quickstart-certificates.adoc

```diff
@@ -0,0 +1,121 @@
+:title: Certificates Quickstart
+:type: quickStart
+:status: published
+:summary: Keystore and certificate instructions.
+:order: 01
+
+=== Certificates (Quick Start)
+
+${branding} comes with a default keystore that contains certificates.
+This allows the distribution to be unzipped and run immediately.
+If these certificates are sufficient for testing purposes, proceed to <<_configuring_quick_start,Configuring (Quick Start)>>.
+
+To test federation using 2-way TLS, the default keystore certificates will need to be replaced, using either the included <<_demo_certificate_authority_ca,Demo Certificate Authority>> or by <<_creating_self_signed_certificates,Creating Self-signed Certificates>>.
+
+If the installer was used to install the ${branding} and a hostname other than "localhost" was given, the user will be prompted to upload new trust/key stores.
+
+If the hostname is `localhost` or, if the hostname was changed _after_ installation, the default certificates will not allow access to the ${branding} instance from another machine over HTTPS (now the default for many services).
+The Demo Certificate Authority will need to be replaced with certificates that use the fully-qualified hostname of the server running the ${branding} instance.
+
```

Comment:
We should probably have something in here at this point that covers the most common scenario when you're setting up DDF and that is when you're handed a PKCS12 keystore to use or you have a private key + cert + CAs that you need to use. We need to cover going from that P12 file (which we sort of cover in managing certificate contents when you are trying to create a new keystore) to what DDF needs. The basics are really: CAs in the truststore (which we say already) and private key + CAs in the keystore (which we don't really say). The keystore part is probably what confuses folks the most, since you generally only need the private key there, but since we use it for other things like verifying signatures, we need the CAs as well.